src/cloud/auth.rs

```diff
@@ -70,12 +72,36 @@ pub async fn do_login(client: &CloudClient) -> anyhow::Result<()> {
     )
 }
 
-pub async fn logout(_c: &options::Logout) -> anyhow::Result<()> {
-    write_json(
-        &cloud_config_file()?,
-        "cloud config",
-        &CloudConfig { access_token: None },
-    )?;
-    print::success("You're now logged out from EdgeDB Cloud.");
+pub fn logout(c: &options::Logout, options: &CloudOptions) -> anyhow::Result<()> {
+    if c.all_profiles {
+        let cloud_creds = cloud_config_dir()?;
+        let dir_entries = match fs::read_dir(cloud_creds.clone()) {
+            Ok(d) => d,
+            Err(e) if e.kind() == io::ErrorKind::NotFound => return Ok(()),
+            Err(e) => anyhow::bail!(e),
+        };
+        for item in dir_entries {
+            let item = item?;
+            if let Ok(filename) = item.file_name().into_string() {
+                write_json(
+                    &cloud_creds.join(filename),
+                    "cloud config",
+                    &CloudConfig { secret_key: None },
```

Comment:
Oh, sorry this was in original code too. But why don't you just drop a file? Why writing a JSON with an empty secret key?

Reply:
Per f2f - this was probably because we had or tried to have more than just `secret_key` in the JSON file in the past. Changed to dropping the file now.